requirements.txt

```diff
@@ -1,2 +1,2 @@
-schwifty==2024.1.1.post0
+schwifty @ git+https://github.com/LN-Zap/schwifty@main
```

Comment:
Lets reference a specific ref / commit hash so that we have better reproducibility.